src/java.base/share/classes/sun/security/ssl/RSAKeyExchange.java

```diff
@@ -292,8 +292,7 @@ class RSAKAKeyDerivation implements SSLKeyDerivation {
             }
 
             @Override
-            public SecretKey deriveKey(String algorithm,
-                    AlgorithmParameterSpec params) throws IOException {
+            public SecretKey deriveKey(String type) throws IOException {
```

Comment:
How about changing the variable name to `typeNotUsed` like `SSLKeyDerivation.LegacyMasterKeyDerivation`?

Reply:
Yes, missed this one.